modal/cli/app.py

```diff
@@ -115,7 +115,7 @@ def logs(
     ```
 
     """
-    app_identifier = warn_on_name_option("stop", app_identifier, name)
+    app_identifier = warn_on_name_option("logs", app_identifier, name)
```

Comment:
Unrelated but happened to run this to verify that normal deprecation warnings are still formatted as we expect and noticed this bug!